assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/components/styles.scss

```diff
@@ -29,4 +29,19 @@ $input-text: 'input[type=' text ']';
 			text-align: left;
 		}
 	}
+
+	&__default-taxes-info {
+		color: var(--ee-color-secondary-super-high-contrast);
+		font-weight: bold;
+		font-size: 1.3em;
+		animation: animate_color 2s infinite;
+	}
+	@keyframes animate_color {
+		from {
+			color: var(--ee-color-secondary-super-high-contrast);
+		}
+		to {
+			color: var(--ee-color-secondary-low-contrast);
+		}
+	}
```

Comment:
@nicolad sorry for entering your territory 😄

Reply:
it's a territory we share with @tn3rb :)